File: MohirDev/Linear-Search/hw-1.py
myList = [1,3,4,6,7,8,10,12,23,45,56,78,99]

def linear_search(n):
    for i in range(len(myList)):
        if myList[i] == n: return -1
    return None

print("index = 7", linear_search(7))

File: MohirDev/Linear-Search/hw-1.cpp
#include <iostream>
using namespace std;

void LinearSearch(int arr[], int size, int target){
    int count = 0;
    for(int i = 0; i < size; i++){
        if(target == arr[i]){
            count++;
        }
    }
    if(count == 0){
        cout<<"yo'q"<<endl;
    }
    else{
        cout<<"bor"<<endl;
    }
    
}

int main(){

    int myList [] = {1,3,4,6,7,8,10,12,23,45,56,78,99};
    int target = 12;
    int size =  sizeof(myList) / sizeof(myList[0]);
    LinearSearch(myList, size, target);
    return 0;
}